src/main/java/io/javalin/plugin/openapi/OpenApiOptions.kt

```diff
@@ -88,6 +89,16 @@ class OpenApiOptions constructor(val initialConfigurationCreator: InitialConfigu
         }
     }
 
+    fun examples(examples: Map<Class<*>, Map<String, Example>>) {
+        openApiExamples = examples.mapValues { it.value.toMutableMap() }.toMutableMap()
+    }
+
+    inline fun <reified T : Any> addExampleForSchema(name: String, example: Example) = addExampleForSchema(T::class.java, name, example)
```

Comment:
I was thinking about the naming and currently it is a little bit inconsistent. That's why I think we should rename it to `addExample`.

Reply:
Will fix.

Reply:
Fixed